addons/frag/functions/fnc_dev_sphereDraw.sqf

```diff
@@ -0,0 +1,52 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add a colored sphere at a specified point
+ *
+ * Arguments:
+ * 0: ASL position to add sphere <ARRAY>
+ * 1: Color of sphere <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [unit0, player] call ace_frag_fnc_dev_switchUnitHandle;
+ *
+ * Public: No
+ */
+params [
+    ["_posASL", [0, 0, 0], [[]], [2,3]],
+    ["_color", "(1,0,0,0.5)", [""]]
+];
+
+if (count _posASL < 3) then
+{
+    _posASL pushBack 0;
+    _posASL = ASLtoATL _posASL;
+    _posASL set [2, 0];
+    _posASL = ATLtoASL _posASL;
+};
+
+if (_color select [0,1] != "(") then
+{
+    switch (toLower _color) do
```

Comment:
```suggestion
    switch (toLowerANSI _color) do
```

Reply:
Sounds good!

Reply:
Addressed in https://github.com/acemod/ACE3/pull/9728/commits/3f45f7735b6b6c50f01d6fcb50db08f20548c0c4